src/components/application_manager/src/commands/mobile/register_app_interface_request.cc

```diff
@@ -73,20 +74,23 @@ mobile_apis::AppHMIType::eType StringToAppHMIType(const std::string &str) {
 }
 
 std::string AppHMITypeToString(mobile_apis::AppHMIType::eType type) {
-  switch(type) {
-    case mobile_apis::AppHMIType::DEFAULT:            return "DEFAULT";
-    case mobile_apis::AppHMIType::COMMUNICATION:      return "COMMUNICATION";
-    case mobile_apis::AppHMIType::MEDIA:              return "MEDIA";
-    case mobile_apis::AppHMIType::MESSAGING:          return "MESSAGING";
-    case mobile_apis::AppHMIType::NAVIGATION:         return "NAVIGATION";
-    case mobile_apis::AppHMIType::INFORMATION:        return "INFORMATION";
-    case mobile_apis::AppHMIType::SOCIAL:             return "SOCIAL";
-    case mobile_apis::AppHMIType::BACKGROUND_PROCESS: return "BACKGROUND_PROCESS";
-    case mobile_apis::AppHMIType::TESTING:            return "TESTING";
-    case mobile_apis::AppHMIType::SYSTEM:             return "SYSTEM";
-    default:
-      NOTREACHED();
-      return NULL;
+  const std::map<mobile_apis::AppHMIType::eType, std::string> appHMITypeMap = {
+    {mobile_apis::AppHMIType::DEFAULT, "DEFAULT"},
+    {mobile_apis::AppHMIType::COMMUNICATION, "COMMUNICATION"},
+    {mobile_apis::AppHMIType::MEDIA, "MEDIA"},
+    {mobile_apis::AppHMIType::MESSAGING, "MESSAGING"},
+    {mobile_apis::AppHMIType::NAVIGATION, "NAVIGATION"},
+    {mobile_apis::AppHMIType::INFORMATION, "INFORMATION"},
+    {mobile_apis::AppHMIType::SOCIAL, "SOCIAL"},
+    {mobile_apis::AppHMIType::BACKGROUND_PROCESS, "BACKGROUND_PROCESS"},
+    {mobile_apis::AppHMIType::TESTING, "TESTING"},
+    {mobile_apis::AppHMIType::SYSTEM, "SYSTEM"}
+  };
+  try {
```

Comment:
@jacobkeeler we use Google Stle guide and is doesn't allow exceptions 
https://google.github.io/styleguide/cppguide.html#Exceptions